src/VCR_InterfaceTasks.cpp

```diff
@@ -15,23 +15,22 @@
 #include "VCR_Globals.h"
 #include "VehicleStateMachine.h"
 
-bool init_read_adc0_task(const unsigned long& sysMicros, const HT_TASK::TaskInfo& taskInfo)
+bool init_read_adc0_task()
 {
-
+    adc_0.init();
```

Comment:
this ADC implementation is broken now. Merge main INTO your test_i2c branch and these will resolve themselves.